src/java/picard/sam/DuplicationMetrics.java

```diff
@@ -41,10 +41,13 @@
      */
     public long UNPAIRED_READS_EXAMINED;
 
-    /** The number of mapped read pairs examined. */
+    /** The number of reads that examined that were either Secondary or Supplementary */
+    public long SECONDARY_OR_SUPPLEMENTARY_READS;
```

Comment:
1. I dropped the _EXAMINED for two reasons:
- it was too long, 
- I want all the different MarkDuplicates to tally this regardless of how they treat Secondary_orSupplementary reads...so I thought that it would be less confusing that way. 
2. As you wish.